packages/expo-localization/ios/LocalizationModule.swift

```diff
@@ -19,6 +19,25 @@ public class LocalizationModule: Module {
     Function("getCalendars") {
       return Self.getCalendars()
     }
+    OnCreate {
+      let enableRTL = Bundle.main.object(forInfoDictionaryKey: "ExpoLocalization_supportsRTL")
+      if let enableRTL = enableRTL as? Bool {
```

Comment:
Sorry for nitpicking 😅 
```suggestion
      if let enableRTL = Bundle.main.object(forInfoDictionaryKey: "ExpoLocalization_supportsRTL") as? Bool {
```

Reply:
Don't be, I want to learn 🙇